biostructbenchmark/__main__.py

```diff
@@ -1,9 +1,28 @@
+"""Single-file validation"""
+
 import argparse
 import os
+from Bio.PDB import *
+from typing import Optional
+import ast
+from cli import arg_parser
```

Comment:
`from . import cli` would be more common for relative script
the call `cli.arg_parser()` instead of `arg_parser()`